css/css-ui/outline-auto-width-001.html

```diff
@@ -6,21 +6,34 @@
 <meta name=flags content="">
 <meta name=assert content="outline-width is ignored when outline-style is auto">
 <link rel=help href="https://drafts.csswg.org/css-ui-4/#outline-style">
-<link rel="match" href="reference/outline-auto-with-001-ref.html">
+<link rel="match" href="reference/outline-auto-width-001-ref.html">
+<link rel="match" href="reference/outline-auto-width-001-b-ref.html">
```

Comment:
Might be good to add a HTML comment explaining that the 2nd ref is for allowing `outline-style: auto` to behave as `none`.

Reply:
```suggestion
<link rel="match" href="reference/outline-auto-width-001-ref.html">
<link rel="match" href="reference/outline-auto-width-001-b-ref.html"><!-- Allowing 'outline-style: auto' to behave the same as 'solid'-->
```